CMakeLists.txt

```diff
@@ -68,7 +68,12 @@ pkg_check_modules(LIBMODULEMD REQUIRED modulemd-2.0>=2.11.2)
 pkg_check_modules(REPO REQUIRED librepo>=1.18.0)
 include_directories(${REPO_INCLUDE_DIRS})
 link_directories(${REPO_LIBRARY_DIRS})
+
 pkg_check_modules(RPM REQUIRED rpm>=4.15.0)
+if (RPM_VERSION VERSION_GREATER_EQUAL "4.19.0")
```

Comment:
as per below and [this bz comment](https://bugzilla.redhat.com/show_bug.cgi?id=2372978#c27), this is wrong. It should be something like 5.99.0.